flowapi/Dockerfile

```diff
@@ -2,12 +2,14 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-FROM python:3.7
+FROM python:3.7-alpine
 
 WORKDIR /flowapi
 COPY Pipfile* /
-RUN pip install pipenv && pipenv install --deploy
+RUN apk update && apk add libzmq && apk add --virtual build-dependencies build-base gcc wget git musl-dev zeromq-dev && \
```

Comment:
Should we use the builder pattern here too (similar to the `flowauth` dockerfile below)? Or not worth the hassle because the resulting image is already fairly small?